_field-types/supported-field-types/star-tree.md

```diff
@@ -23,10 +23,10 @@ To use a star-tree index, follow the instructions in [Enabling a star-tree index
 
 ## Limitations
 
-The star-tree index feature has the following limitations:
+The STIX feature has the following limitations:
```

Comment:
We can either have all the limitations here or we can link up the limitations section in the other page.

Reply:
Let's move all the limitations here.